handbook/engineering/engineering-management.md

```diff
@@ -49,20 +49,52 @@ Here are some books that multiple people on the team have read and recommend. If
 - [The Five Dysfunctions of a Team](https://www.amazon.com/Five-Dysfunctions-Team-Leadership-Fable/dp/0787960756) - A model of how to think about what makes a highly effective team paired with a fictional story that illustrates the model.
 - [The Leader’s Guide to Unconscious Bias](https://www.amazon.com/Leaders-Guide-Unconscious-Bias-High-Performing/dp/1982144319) ([facilitated small group discussions](../support/support-values-enablement.md#part-2-deep-dive-on-be-welcoming-and-inclusive).
 
-## Temporary Management Positions
+## Promotion requests
 
-We create temporary management positions when there is an organizational need. These may be filled by someone who is transitioning into the role (Interim Manager), experimenting with the role as they work on determining their career path (Acting Manager) or, someone who is just filling in and who is not interested in pursuing an Engineering Manager role long term (Acting Manager). This difference should be made explicit with the individual and team members before the temporary role is assigned.
+To be documented.
 
-### Acting Manager
+## Compensation increase requests
 
-An Acting Manager occupies a role temporarily and will move back to their original position after a set amount of time or other conditions. An Acting Manager may be experimenting with the role as a part of determining their career development path or could be filling in for a vacant role while we hire.
+To be documented.
 
-### Interim Manager
+## Temporary management positions
```

Comment:
Do we have a past example of a temporary management position? I am wondering if we can/should delete this bit and just focus the docs on how we train new managers.

Reply:
We have acting managers, which are temporary management positions all the time. However, I don't think it adds value in this context right now so will remove it.